recirq/otoc/loschmidt/tilted_square_lattice/end-to-end-test.sh

```diff
@@ -0,0 +1,3 @@
+python gen-small-v1.py
```

Comment:
Worth a comment to explain this file?  Also, does this file need license info?

Reply:
added.

Worth noting that I have much more robust plans for doing end-to-end (ie integration testing) once more of the benchmarks are in place; but I figured it wouldn't hurt to have this baby bash script for the time being.